doc/source/how-to/continuous-integration.rst

```diff
@@ -176,3 +176,356 @@ Workflow examples are provided for checking :ref:`Coding style`,
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **read:gpg_key**                                                                    | |:x:|                  |
+  |  Read public user GPG keys.                                                         |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **admin:ssh_signing_key**                                                           | |:x:|                  |
+  |  Full control of public user SSH signing keys.                                      |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **write:ssh_signing_key**                                                           | |:x:|                  |
+  |  Write public user SSH signing keys.                                                |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **read:ssh_signing_key**                                                            | |:x:|                  |
+  |  Read public user SSH signing keys.                                                 |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+
+.. _pypi_tokens:
+
+PyPI tokens
+~~~~~~~~~~~
+
+.. _pypi_private_token:
+
+``PYANSYS_PYPI_PRIVATE_PAT``
+****************************
+
+The ``PYANSYS_PYPI_PRIVATE_PAT`` token is used for authentication when uploading
+libraries to the private Ansys PyPI index. This token can be used as the password for
+the `twine <https://twine.readthedocs.io/en/stable/>`_ library.
+
+.. _pypi_token:
+
+``PYPI_TOKEN``
+**************
+
+The value of the ``PYPI_TOKEN`` token is unique for each repository. 
+This token is used for authentication when uploading libraries to the public PyPI index.
+
+
+``PYPI_TESTING_TOKEN``
+**********************
+
+The ``PYPI_TESTING_TOKEN`` token is used for testing uploads to the public PyPI index.
+
+
+Other tokens
+~~~~~~~~~~~~
+
+``HUGO_THEME_TOKEN``
+********************
+
+The ``HUGO_THEME_TOKEN`` token is used to ... # todo: to be added
+
+
+``TWINE_TOKEN``
+***************
+
+The ``TWINE_TOKEN`` token is used for authentication when uploading libraries to PyPI.
+Its value might change
+across repositories.
+Depending if the library is uploaded to a public PyPI index or the Ansys private PyPI index, its value matches
+:ref:`pypi_private_token` or :ref:`pypi_token`.
+
+
+
+.. _organization_bot:
+
+Organization bot
+----------------
+
+Because the usage of personal access tokens (PATs) is discouraged, the ``Ansys``
+and ``Ansys-internal`` organizations provide a bot named ``ansys-bot`` to
```

Comment:
```suggestion
Because the usage of personal access tokens (PATs) is discouraged, the ``ansys``
and ``ansys-internal`` organizations provide a bot named ``ansys-bot`` to
```